generators/module/templates/default/tests/_partials/SolutionX.ModuleTypeX.ModuleNameX.Tests.csproj.9.3.0.partial

```diff
@@ -277,7 +279,13 @@
     <Content Include="App.config" />
   </ItemGroup>
   <ItemGroup>
-    <Analyzer Include="..\..\..\packages\xunit.analyzers.0.8.0\analyzers\dotnet\cs\xunit.analyzers.dll" />
+    <Analyzer Include="..\..\..\packages\xunit.analyzers.0.10.0\analyzers\dotnet\cs\xunit.analyzers.dll" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\code\<%= solutionX %>.<%= moduleTypeX %>.<%= moduleNameX %>.csproj">
```

Comment:
Please double check if we need this. thx

Reply:
Removed